src/main/java/frc/team2412/robot/Subsystems/LiftSubsystem.java

```diff
@@ -0,0 +1,42 @@
+package frc.team2412.robot.Subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team2412.robot.Subsystems.constants.LiftConstants.LiftState;
+
+public class LiftSubsystem extends SubsystemBase {
+
+	// creates a DoubleSolenoid to use, mainly for ease
+	private DoubleSolenoid m_liftUpDown;
+
+	// Sets the state of the lift to down because of init it is down
+	private LiftState m_currentState = LiftState.DOWN;
+
+	// For when we create a liftSubsystem, it makes a liftSubsystem which uses our
+	// local liftSubsystem
+	public LiftSubsystem(DoubleSolenoid liftUpDown) {
+		this.m_liftUpDown = liftUpDown;

```

Comment:
Can m_currentState be updated to the actual value if it does not match the hardcoded value?